nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -43,26 +45,47 @@
 __all__ = ['LevelPruner', 'L1NormPruner', 'L2NormPruner', 'FPGMPruner', 'SlimPruner', 'ActivationPruner',
            'ActivationAPoZRankPruner', 'ActivationMeanRankPruner', 'TaylorFOWeightPruner']
 
+NORMAL_SCHEMA = {
+    Or('sparsity', 'sparsity_per_layer'): And(float, lambda n: 0 <= n < 1),
+    SchemaOptional('op_types'): [str],
+    SchemaOptional('op_names'): [str]
+}
+
+GLOBAL_SCHEMA = {
+    'total_sparsity': And(float, lambda n: 0 <= n < 1),
+    SchemaOptional('max_sparsity_per_layer'): And(float, lambda n: 0 < n <= 1),
```

Comment:
if there is `max_sparsity_per_layer`, why not add `min_sparsity_per_layer`?

Reply:
good idea, we can discuss this and support it in the following pr.